.github/workflows/AutoAssignToSprint.yml

```diff
@@ -0,0 +1,18 @@
+name: Auto Assign to Sprint Project
+
+on:
+  issues:
+    types: [opened]
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  assign_one_project:
+    runs-on: ubuntu-latest
+    name: Assign to One Project
+    steps:
+    - name: Assign NEW issues to sprint proect
+      uses: srggrs/assign-one-project-github-action@1.2.1
+      if: github.event.action == 'opened'
```

Comment:
do you need this? 

I think this always evaluates to true. on L5 you already specified the trigger is that an issue is opened to be the action. 

I think the example you followed only did it because it was also being triggered when someone added a label

Reply:
just the if statement on L16? I assume we still need the 'uses' on L15?

Reply:
yep